### PR TITLE
Replace 'inline extern' with 'fixed function block'

### PR DESCRIPTION
**Pull request by jfingerh**
The term 'inline extern' is a misleading choice of words for this functionality in PNA, and fixed function block has a longer history of being used for such things.

Remove a left-over mention of the pre control that was missed before.

Correct the name of the extern function set_entry_expire_time_if in some examples in comments.

Use INCLUDE directive in a few more places of PNA.mdk that will help keep the code snippets from the pna.p4 include file consistent between that file and the spec more easily in the future.

### CODE REVIEW

**Comment by jfingerh** [comment]
@mariobaldi These are mostly minor fixes, that I think would be reasonable to merge if you approve, without going through meeting discussions on.

**Review by jnfoster — APPROVED** [review]
I have been a proponent of this terminology, so I wholeheartedly approve.

File: pna.p4 [hunk 1]
```diff
@@ -793,13 +797,13 @@ extern void update_expire_info(
...
 //
 // Examples:
-// set_expire_time_if(hdr.tcp.flags == TCP_FLG_SYN &&
-//                    meta.direction == OUTBOUND,
-//                    tcp_connection_start_time_profile_id);
-// set_expire_time_if(hdr.tcp.flags == TCP_FLG_ACK,
-//                    tcp_connection_continuation_time_protile_id);
-// set_expire_time_if(hdr.tcp.flags == TCP_FLG_FIN,
-//                    tcp_connection_close_time_profile_id);
+// set_entry_expire_time_if(hdr.tcp.flags == TCP_FLG_SYN &&
```
Comment (jnfoster) [inline pna.p4:800]: Were these changes intentional?

File: pna.p4 [hunk 2]
```diff
@@ -793,13 +797,13 @@ extern void update_expire_info(
...
+//                          meta.direction == OUTBOUND,
+//                          tcp_connection_start_time_profile_id);
+// set_entry_expire_time_if(hdr.tcp.flags == TCP_FLG_ACK,
```
Comment (jnfoster) [inline pna.p4:803]: Same.

File: pna.p4 [hunk 3]
```diff
@@ -793,13 +797,13 @@ extern void update_expire_info(
...
+//                          tcp_connection_continuation_time_protile_id);
+// set_entry_expire_time_if(hdr.tcp.flags == TCP_FLG_FIN,
```
Comment (jnfoster) [inline pna.p4:805]: Same.

File: pna.p4 (hunk 1 above)
Comment (jfingerh) [reply pna.p4:800]: Yes.  The name set_expire_time_if in the comments is not defined in the pna.p4 header file.  set_entry_expire_time_if is defined.

File: PNA.mdk [hunk 4]
```diff
@@ -1057,7 +1056,7 @@ existing slides.
 
 ## Is it inefficient to have the `MainParser` redo work? {#appendix-reparsing}
 
-If the only changes made by the inline extern in the network-to-host
+If the only changes made by the decryption fixed function block in the network-to-host
```
Comment (mariobaldi) [inline PNA.mdk:1059]: In the architecture picture and description what was previously called "inline extern" is now called "inline accelerators". 
This said, the text also states that "inline accelerators are fixed-function blocks", so even though we use different terms for the same element, the terminology seems coherent to me. But I just wanted to bring this to your attention to make sure you are aware of and fine with it.

**Review by mariobaldi — APPROVED** [review]
Thank you for catching these inconsistencies and making these improvements.

File: PNA.mdk (hunk 4 above)
Comment (jafingerhut) [reply PNA.mdk:1059]: Good point.  It seems worth having a short glossary of terms somewhere, along with preferred terms to use in the spec for consistency's sake.  I will merge this one as is, and create a separate issue about the glossary and whether to use "inline accelerator" in most places in the PNA spec.
